### PR TITLE
Resolve warn: postfix operator should be explicitly enabled

### LINKED ISSUE #685: Compile Warning in OapRpcManagerSlave
@gczsjdy can you check this issue?
```
[WARNING] /Users/linhongl/IdeaProjects/OAP/src/main/scala/org/apache/spark/sql/oap/rpc/OapRpcManagerSlave.scala:85: postfix operator += should be enabled
by making the implicit value scala.language.postfixOps visible.
Warning:(85, 48) postfix operator += should be enabled
by making the implicit value scala.language.postfixOps visible.
This can be achieved by adding the import clause 'import scala.language.postfixOps'
or by setting the compiler option -language:postfixOps.
See the Scaladoc for value scala.language.postfixOps for a discussion
why the feature should be explicitly enabled.
    getMaterials.foreach(oapHeartbeatMaterials +=)
```

### PR DESCRIPTION
**Pull request by gczsjdy**
## What changes were proposed in this pull request?

To resolve #685 .

## How was this patch tested?

Existing UTs.

### CODE REVIEW

File: src/main/scala/org/apache/spark/sql/oap/rpc/OapRpcManagerSlave.scala [hunk 1]
```diff
@@ -82,6 +82,7 @@ private[spark] class OapRpcManagerSlave(
   }
 
   private[spark] def registerHearbeat(getMaterials: Seq[() => Heartbeat]): Unit = {
+    import scala.language.postfixOps
     getMaterials.foreach(oapHeartbeatMaterials +=)
```
Comment (adrian-wang) [inline src/main/scala/org/apache/spark/sql/oap/rpc/OapRpcManagerSlave.scala]: I think we can just use
```
 getMaterials.foreach(oapHeartbeatMaterials += _)
```

Comment (gczsjdy) [reply src/main/scala/org/apache/spark/sql/oap/rpc/OapRpcManagerSlave.scala]: Both ways are graceful, : )

Comment (gczsjdy) [reply src/main/scala/org/apache/spark/sql/oap/rpc/OapRpcManagerSlave.scala]: Ok, I like your suggestion more.

**Comment by linhong-intel** [comment]
LGTM.
checked the test log. no warning any more.

**Comment by gczsjdy** [comment]
Thanks @linhong-intel @adrian-wang

**Pull request merged**